packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -707,6 +706,31 @@ Or read our docs at https://www.prisma.io/docs/concepts/components/prisma-client
       })
     }
 
+    /**
+     * Executes a raw command only for MongoDB
+     *
+     * @param command
+     * @returns
+     */
+    $runCommandRaw(command: object) {
+      if (config.activeProvider !== 'mongodb') {
+        throw new PrismaClientValidationError(`The ${config.activeProvider} provider does not support $runCommandRaw`)
```

Comment:
```suggestion
        throw new PrismaClientValidationError(`The ${config.activeProvider} provider does not support $runCommandRaw. Use the mongodb provider.`)
```

Also, is there a test for this?